.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
```

Comment:
Instead of extending the configurations from @mate-academy/linthtml-config, you have completely overwritten it. You should extend the initial configuration and only add/update the configurations specific to your project. 🚀

Reply:
Sorry, but i don't understand how i should do it in the right way

Reply:
perhaps you don't even need to change it at all